.github/workflows/tests-integration.yml

```diff
@@ -1,5 +1,4 @@
-# Executes the integration tests on all platforms
-name: Integration
+name: Tests
```

Comment:
Minor nit, but we should also rename this file as it is no longer just integration tests. Also, any reason why we can't combine this with run-tests.yml?

Reply:
`run-tests.yml` is the workflow being re-used for the various test/os combinations that we have here.